config.json

```diff
@@ -1268,6 +1268,19 @@
             "sets"
         ]
     },
+    {
+        "uuid": "b0c7cf95-6470-4c1a-8eaa-6775310926a2",
+        "slug": "spiral-matrix",
+        "core": false,
+        "unlocked_by": null,
+        "difficulty": 2,
+        "topics": [
+            "algorithms",
+            "control-flow",
+            "lists",
+            "logic"
```

Comment:
I don't think that `logic` really applies to this exercise. A better example would be the [`zebra-puzzle`](https://github.com/exercism/python/tree/master/exercises/zebra-puzzle) exercise.

Reply:
Good point, @cmccandless. I decided to delete "logic" from the list of topics.